<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VpncProp</class>
 <widget class="QWidget" name="VpncProp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cisco VPN</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>General</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="textLabel3">
        <property name="text">
         <string>Gateway:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="buddy">
         <cstring>leGateway</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="KLineEdit" name="leGateway"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelGroup">
        <property name="text">
         <string>Group name:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="buddy">
         <cstring>leGroupName</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="KLineEdit" name="leGroupName"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>User &amp;Password</string>
        </property>
        <property name="buddy">
         <cstring>leUserPassword</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="KLineEdit" name="leUserPassword">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="passwordMode">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cboUserPassOptions">
          <item>
           <property name="text">
            <string>Always Ask</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Store</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Not Required</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>&amp;Group Password:</string>
        </property>
        <property name="buddy">
         <cstring>leGroupPassword</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="KLineEdit" name="leGroupPassword">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="passwordMode">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cboGroupPassOptions">
          <item>
           <property name="text">
            <string>Always Ask</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Store</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Not Required</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="1">
       <widget class="QCheckBox" name="cbShowPasswords">
        <property name="text">
         <string>&amp;Hide passwords</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Optional</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>&amp;Username:</string>
        </property>
        <property name="buddy">
         <cstring>leUserName</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="KLineEdit" name="leUserName"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>&amp;Domain:</string>
        </property>
        <property name="buddy">
         <cstring>leDomain</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="KLineEdit" name="leDomain"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>&amp;Encryption Method:</string>
        </property>
        <property name="buddy">
         <cstring>cboEncryptionMethod</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="cboEncryptionMethod">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text">
          <string>Secure</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Weak</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>None</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>&amp;NAT Traversal:</string>
        </property>
        <property name="buddy">
         <cstring>cboNatTraversal</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="cboNatTraversal">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text">
          <string>Cisco UDP</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>NAT-T</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Disabled</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>IKE DH Group</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="cboDHGroup">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text">
          <string>DH Group 1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>DH Group 2 (default)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>DH Group 5</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QCheckBox" name="chkDeadPeerDetection">
        <property name="text">
         <string>Enable Dead &amp;Peer Detection</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>cbShowPasswords</sender>
   <signal>toggled(bool)</signal>
   <receiver>leGroupPassword</receiver>
   <slot>setPasswordMode(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>236</x>
     <y>181</y>
    </hint>
    <hint type="destinationlabel">
     <x>234</x>
     <y>144</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbShowPasswords</sender>
   <signal>toggled(bool)</signal>
   <receiver>leUserPassword</receiver>
   <slot>setPasswordMode(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>331</x>
     <y>169</y>
    </hint>
    <hint type="destinationlabel">
     <x>313</x>
     <y>108</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
